irods/test/ticket_test.py

```diff
@@ -71,6 +73,27 @@ def tearDown(self):
                 ses.users.remove(u)
 
 
+    def test_admin_keyword_for_tickets (self):
+
+        N_TICKETS = 3
+
+        # create some tickets as alice
+
+        with self.login(self.alice) as alice:
+            alice_home_path = self.irods_homedir(alice, path_only = True)
+            ticket_strings = [ Ticket(alice).issue('read', alice_home_path).string for _ in range(N_TICKETS) ]
+
+        # use admin flag to delete alice's tickets
+
+        with helpers.make_session() as ses:
+            alices_tickets = [t[TicketQuery.Ticket.string] for t in ses.query(TicketQuery.Ticket).filter(TicketQuery.Owner.name == 'alice')]
+            self.assertEqual(len(alices_tickets),N_TICKETS)
+            for s in alices_tickets:
+                Ticket( ses, s ).delete(**{kw.ADMIN_KW:''})
+            alices_tickets = [t[TicketQuery.Ticket.string] for t in ses.query(TicketQuery.Ticket).filter(TicketQuery.Owner.name == 'alice')]
+            self.assertEqual(len(alices_tickets),0)
+
+
```

Comment:
is this test demonstrating deleting another user's (alice) tickets using a different iRODS account (i.e. rods)?

does `helpers.make_session()` represent a rodsadmin user?
if so, let's add some comments stating that. it's not clear who is deleting the tickets.

Reply:
@korydraughn  Yes, did add the comments, but it probably needs to be documented more generally how these tests are to be run.  Whether in CI or standalone, irods.test.helpers.make_session( ) tends to be used to create the default "from environment" session with an account iinit'ed for rods.